docs/DeveloperGuide.md

```diff
@@ -81,6 +81,16 @@ The list medication command extends from Command parent class and contains the f
 then automatically checks off the relevant medication in the DailyMedication list, without user specifying 
 `(-m/-a/-e)` **after** `take -l [index]`.
 
+### View Medication Command
+The view medication command extends from Command parent class and contains the following:
+
+Given below is an example of and how the view command behaves at each step 
+with the help of a the sequence diagram with a usage scenario.
+
+![ViewCommand.png](images/ViewCommand.png)
+- Step 1. User initiates MediTracker and done a viewCommand.
```

Comment:
"done" should be "does" right?

Reply:
Yes, you are right. Good catch. It is resolved in [191f06a](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/242/commits/191f06a09a9dce0878d824a4640aa440d35e32ca) , but still in progress.